package.json

```diff
@@ -62,10 +62,11 @@
     "babel-polyfill": "^6.9.0",
     "babel-preset-es2015": "^6.3.13",
     "eslint-if-supported": "^1.0.1",
-    "feathers": "^2.0.0",
+    "feathers": "^2.0.3",
+    "feathers-socketio": "^1.4.2",
```

Comment:
It's only in development for running the unit tests. I'll try with out it, but pretty sure socket mixins aren't run unless this is present in the test app. I haven't seen any weirdness.